sympy/core/numbers.py

```diff
@@ -16,7 +16,8 @@
     as_int, integer_types, long, string_types, with_metaclass, HAS_GMPY,
     SYMPY_INTS, int_info)
 from sympy.core.cache import lru_cache
-
+from sympy.external import import_module
+nums = import_module('numbers')
```

Comment:
Is there a reason this isn't just `import numbers`?

Reply:
Yep, it fails on 2.7 due to name collision.

Reply:
Oh I see. I think it should work if you add `absolute_import` to the `__future__` import at the top of the file.